README.md

```diff
@@ -138,10 +138,10 @@ $ pre-commit install
 $ poetry run pytest
 ```
 
-Running the tests in a docker container containing an ohsome API instance is faster, but not mandatory. To set up and start such a docker container run the following command before running the tests.
+Running the tests against a dockerised local instance of the ohsome API is faster, but not mandatory. By default the test will issue a warning that it could not find the local instance and then fall back to the public one. To set up and start a local instance run the following command before running the tests (see https://github.com/GIScience/ohsome-api-dockerized).
 
 ```
-$ docker run -dt --name ohsome-api -p 8080:8080 julianpsotta/ohsome-api:1.3.2
+$ docker run -dt --name ohsome-api -p 8080:8080 julianpsotta/ohsome-api:latest
```

Comment:
Better keep the version number 1.3.2. See comment by @MichaelsJP on PR #41: 

> Better to use the detached mode here with the latest version instead of latest. Latest will be the repository latest. Mostly nightly builds etc.

Reply:
accroding to https://github.com/GIScience/ohsome-api-dockerized/issues/32 we can now use latest. Favour to change the README?

Reply:
thanks!